assets/style.sass

```diff
@@ -75,6 +75,9 @@ section
 .category
   margin-left: 1in
 
+.gig
+  break-inside: avoid
```

Comment:
I played with something similar, but now that I have added a projects array to my Test Double gig, it is being pushed off the front page.

I am going to hold off on a PR about this until this is merged or closed or we get a chance to pair on it.

I love the way cobbler works, but making a printable paged document from HTML ls always a headache.

I have some ideas, but I will express them in code when/if I have a chance to work on this.